tests/base_test_case.py

```diff
@@ -115,9 +115,9 @@ def assertRaisesRegexp(self, expected_exception, expected_regexp,
             self.fail("Expected exception " + str(expected_exception) +
                       " not raised" + msg)
 
-    def assertLazyMessage(self, assert_function, actual, expected, msg_func):
+    def assertLazyMessage(self, msg_func, assert_function, *args, **kwargs):
```

Comment:
We could probably pass *args and **kwargs into msg_func and lose all the silly lambdas I created. I apparently did not have my python hat on when I wrote this originally. If you want to push without this change and file a ticket, I can worry about it later.